cql3/expr/expression.hh

```diff
@@ -67,6 +67,11 @@ struct column_value {
     column_value(const column_definition* col, ::shared_ptr<term> sub) : col(col), sub(sub) {}
 };
 
+/// A tuple of column selectors. Usually used for range constraints on clustering keys.
+struct column_value_tuple {
+    std::vector<column_value> elements;
+};
```

Comment:
Do you forsee this type ever needing to be contain anything except these "elements"?
If not, maybe it should be just an alias (using column_value_tuple = std::vector<column_value>)?
Or inheritence: struct column_value_tuple : public std::vector<column_value> ?
It could save you a lot of places typing the word "elements".

Reply:
I don't mind a little extra typing if it brings some clarity. And I say that as a slow typist.

Reply:
Or rather, I type that as a slow typist.

A slow typist who likes strong types!

Reply:
I think it's fine, just like `conjunction` has a single member `children`.